kotlin-checks-test-sources/src/main/kotlin/checks/UnnecessaryImportsCheckSamplePartialSemantics.kt

```diff
@@ -2,8 +2,8 @@ package checks
 
 import checks.ClassInSameFileA
 import checks.DelicateCoroutinesApi
-import com.google.common.collect.ImmutableList
-import com.google.common.collect.ImmutableList.copyOf
+import com.google.common.collect.ImmutableList // Noncompliant
+import com.google.common.collect.ImmutableList.copyOf // Noncompliant
```

Comment:
Tnbis is the only issue affected, but those 2 are TP. But only on Linux/MacOS (Windows build is failing because of this change). Don't know what to do since this is a partial semantic test maybe we can accept this, or from these lines.

Reply:
Will be fixed by #447